sql-to-dbsp-compiler/lib/sqllib/src/timestamp.rs

```diff
@@ -50,20 +51,30 @@ impl Debug for Timestamp {
 
 /// Serialize timestamp into the `YYYY-MM-DD HH:MM:SS.fff` format, where
 /// `fff` is the fractional part of a second.
-impl Serialize for Timestamp {
-    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+impl SerializeWithContext<SqlSerdeConfig> for Timestamp {
```

Comment:
should the comment above be changed too?

Reply:
yep